### ISSUE TITLE
Ladder request

### ISSUE THREAD

**Issue by ralstad** [TobbyTheBobby/TimberbornMods#10]
Can you make the ladder mod cost a separate file so a player can adjust the cost for materials needed to build the ladder

**Comment by TobbyTheBobby** [comment]
If you want I can give you something wich is WIP and unstable, but should be stable for your case.

**Comment by ralstad** [comment]
What does wip mean

**Comment by TobbyTheBobby** [comment]
It means Work In Progress (WIP). It means it is not in a state in which I would release it, but it does is in a state where its useable.

**Comment by ralstad** [comment]
Sure I would be willing to try it, it will allow me to change the cost to build it?

John Ladd
JDL Maintenance
Phone #: 317-531-2943
Email: ***@***.*** 

    On Monday, February 26, 2024 at 12:37:05 AM EST, Tobbert ***@***.***> wrote:  
 
 


It means Work In Progress (WIP). It means it is not in a state in which I would release it, but it does is in a state where its useable.

**Comment by ralstad** [comment]
Where you still willing to do that adjusted mod for me?

**Comment by TobbyTheBobby** [comment]
[Here](https://github.com/TobbyTheBobby/TimberbornMods/files/14453909/DifficultySettingsChanger.zip)
you have the build. I forgot about it, thanks for the reminder :)

**Comment by ralstad** [comment]
That says its the difficulty settings changer

**Comment by TobbyTheBobby** [comment]
Yes exactly, as I said a WIP project which does what you want. It is just another mod which im working on :)

**Comment by ralstad** [comment]
How do I change the cost of Ladder in this mod

**Comment by TobbyTheBobby** [comment]
oh ofcourse, i should explain then. Go ingame, then do ctrl d with ladder mod installed aswell. Then search for ladder and should be in there. Then open building cost property and open the values to change the number. press enter to save and then reload and should work. I havent tested properly ofcourse (WIP) but might work. Otherwise I sadly will have to keep improving that mod to release it as it will not be part of ladder itself.

**Comment by ralstad** [comment]
Ok,

Let me make sure I have this correct.

Do i drop the initial unzipped folder into the mods folder of Difficulty settings changer and Tobby tools folders into the mod fold?

Do these run in tbmpl or timberapi?

The run timber to add mods
exit timber
run timber again then do ctrl d

what file do I look for and in what folder?

In whatever file look for what I want to mod and change variables.

Correct?
